units/systemd-journal-upload.service.in

```diff
@@ -14,6 +14,7 @@ After=network-online.target
 [Service]
 ExecStart=@rootlibexecdir@/systemd-journal-upload --save-state
 User=systemd-journal-upload
+Group=systemd-journal-upload
```

Comment:
Yeah, it is redundant. I've add the line just for the consistency to other units. If you do not like it, I will remove it.